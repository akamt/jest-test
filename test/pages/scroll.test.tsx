import React from 'react'
import { render } from '../testUtils'
import ScrollPage from '../../src/pages/scroll'

describe('Scroll page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<ScrollPage />, {})
    expect(asFragment()).toMatchSnapshot()
  })

  it('clicking button triggers alert', () => {
    const { getByText } = render(<ScrollPage />, {})

    const sectionText = getByText('Section1').textContent
    expect(sectionText).toBe('Section1')
  })
})
