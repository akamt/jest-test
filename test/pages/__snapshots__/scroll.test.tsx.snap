// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="h-screen w-screen"
  >
    <nav
      class="flex h-16 justify-center"
    >
      <a>
        Section1
      </a>
    </nav>
    <section
      class="h-1/2 flex justify-center items-center"
    >
      50% height section
    </section>
    <section
      class="h-full flex justify-center items-center"
      id="section1"
    >
      section1!!
    </section>
  </div>
</DocumentFragment>
`;
